recipes/phenoshift/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "phenoshift" %}
+{% set version = "0.0.4.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f451ffc36ce85649b99e64ecd56b8b26e7f10a6587556b1b9eb2670f18d1d96e
+
+build:
+  number: 0
```

Comment:
This should be a noarch python package

Reply:
Agreed, I had this before but the linter doesn't like it. Is the way I am writing it incorrect?